rustls/src/msgs/message_test.rs

```diff
@@ -81,12 +81,6 @@ fn alert_is_not_handshake() {
     assert!(!m.is_handshake_type(HandshakeType::ClientHello));
 }
 
-#[test]
-fn alert_is_not_opaque() {
-    let m = Message::build_alert(AlertLevel::Fatal, AlertDescription::DecodeError);
-    assert!(Message::try_from(m).is_ok());
```

Comment:
`try_from()` does an identity conversion here. I don't think this is testing anything meaningful at this point?